pallets/parachain-staking/src/tests.rs

```diff
@@ -55,25 +55,104 @@ fn invalid_root_origin_fails() {
 #[test]
 fn set_total_selected_event_emits_correctly() {
 	ExtBuilder::default().build().execute_with(|| {
-		assert_ok!(Stake::set_total_selected(Origin::root(), 6u32));
-		assert_last_event!(MetaEvent::Stake(Event::TotalSelectedSet(5u32, 6u32)));
+		// before we can bump total_selected we must bump the blocks per round
+		assert_ok!(ParachainStaking::set_blocks_per_round(Origin::root(), 6u32));
```

Comment:
I do not understand why there should be any changes in any thing related to parachain-staking